src/screens/AttendanceScreen/components/AttendanceInput.tsx

```diff
@@ -0,0 +1,57 @@
+import { Dispatch, SetStateAction } from 'react';
+import { Box, Input as ChakraInput, Grid } from '@chakra-ui/react';
+import theme from '@styles/theme';
+import TransparentInput from './TransparentInput';
+
+interface AttendanceInputProps {
+  answer: string;
+  answerList: string[];
+  setAnswer: Dispatch<SetStateAction<string>>;
+}
+
+const AttendanceInput = ({
+  answer,
+  answerList,
+  setAnswer,
+}: AttendanceInputProps) => {
+  const handleBgColor = (index: number) => {
+    if (index === 0) return '#FFFFFF1A';
+    return answer.length > index ? '#FFFFFF1A' : '#FFFFFF33';
```

Comment:
```suggestion
  const handleBgColor = (answerLength: number ,index: number) => {
    if (index === 0) return '#FFFFFF1A';
    return answerLength > index ? '#FFFFFF1A' : '#FFFFFF33';
   }
```

이렇게 생각했습니다.
이렇게 변경하고 나면, handleBgColor 함수는 더이상 answer 가 있어야만 동작하는 함수가 아니라, param을 주입하기만 하면 동작하는, 순수함수가 되니깐요!

Reply:
아하 **handleBgColor** 함수 말씀하신거였군요 ㅎㅎ 순수함수 깔끔하네요! 수정하겠습니당~!